Add link to AsciiDoc variant on the main page

vpavic opened the pull request:
This commit adds link to AsciiDoc variant on the main page, in order to make it more discoverable.

vpavic committed:
Add link to AsciiDoc variant on the main page

This commit adds link to AsciiDoc variant on the main page, in order to make it more discoverable.

klaude commented:
Thanks for the patch! The deploy preview is failing because hugo can't build the site, due to missing files. 

```
2:27:37 PM: $ hugo
2:27:37 PM: Building sites … ERROR 2022/06/07 19:27:37 [en] REF_NOT_FOUND: Ref "/version/2/1/code_of_conduct.adoc": "/opt/build/repo/content/_index.md:38:32": page not found
2:27:37 PM: WARN 2022/06/07 19:27:37 found no layout file for "asciidoc" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
<snip>
```

Version 2.1 doesn't have an asciidoc variant. Would you be willing to make one and include it in this PR? Thanks!

vpavic commented:
Ah, sorry about that - I didn't build the site locally before submitting the PR.

I'll look into adding the missing version soon, thanks for the feedback.

vpavic commented:
I've fixed the issue. The problem weren't missing files (AsciiDoc variants are apparently generated dynamically) but rather me referencing `.adoc` instead of `.md` in the link. Anyway, the preview is now successfully deployed.

One final thing to consider: I added the link to the AsciiDoc variant as the last one, but my feeling is that Markdown and AsciiDoc variant are probably the most interesting ones to the community and should be the top two in the list.

CoralineAda reviewed (approved):
Thank you for this!
